admin/src/js/controllers/edit-user.js

```diff
@@ -254,24 +279,63 @@ angular
       return hasPlace && hasContact;
     };
 
+    const validateFacilityHierarchy = () => {
+      const placeIds = $scope.editUserModel.place;
+
+      if (!placeIds || placeIds.length === 1) {
```

Comment:
Is this _always_ an array?

Reply:
I believe so because the html uses a multi-select